src/integration/java/io/pinecone/integration/dataPlane/UpdateFetchAndQueryServerlessTest.java

```diff
@@ -123,6 +74,18 @@ public void updateAllParamsFetchAndQuerySyncTest() throws InterruptedException {
                     namespace);
         }
 
+        // wait sometime for the vectors to be upserted
```

Comment:
Upserting required + optional parameters before all of the tests, so we only have to wait 2 minutes for allowing data to upsert once.